data/json/items/comestibles/spice.json

```
@@ -82,6 +82,19 @@
     "//": "a tbps is 4.2g of sugar, meaning the volume needed to be adjusted.",
     "charges": 71
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "sprinkles",
+    "name": "sprinkles",
+    "name_plural": "sprinkles",
+    "weight": 4,
+    "copy-from": "sugar",
+    "container": "bottle_plastic_small",
+    "description": "Known as nonpareils, sprinkles, jimmies, sugar strands or 'hundreds-and-thousands', these colorful balls, rods or flakes of sugar and starch are intended for use decorating soft surfaces of sweets.  Like sugar, they are bad for your teeth and surprisingly not very tasty on their own.",
```

```suggestion
    "description": "Known as nonpareils, sprinkles, jimmies, sugar strands or 'hundreds-and-thousands', these colorful balls, rods or flakes of sugar and starch are intended for decorating soft surfaces of sweets.  Like sugar, they are bad for your teeth and surprisingly not very tasty on their own.", 
```
Also I believe all these synonyms you mentioned are known only to English-speakers and probably won't make any sense to all other languages.

---

That's... a shame.  I don't really know what to do with this.

---

'Sprinkles' are definitely known, but not the others. I'd cut down on the list, leaving only sprinkles.